src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2641,6 +2641,74 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    // Method Description:
+    // - Displays a dialog stating the "Touch, Keyboard and Handwriting Panel
```

Comment:
I mean if you're going to make up a comma for the first one.... why not throw in the Oxford comma too? :P